### PR TITLE
Omniture for R2 page pressing

### PR DESCRIPTION
**Pull request by jennysivapalan**
Follows https://github.com/guardian/frontend/pull/11396
- Ensures Identity references, `<scripts>` and `<noscripts>` are removed.
- Adds a reference to a call an external script see: https://github.com/guardian/frontend/pull/11421 to add page view for Omniture.
- As suggested by @rich-nguyen the original html should be pushed to a new bucket in case we need to re-do this in the future.
- Adds a switch to push the HTML file to a bucket.

Note the changes in https://github.com/guardian/frontend/pull/11421 is not quite available to use but this requires a Fastly change that I'll do in the new year. :fireworks: 

cc/ @JustinPinner

### CODE REVIEW

**Comment by jennysivapalan** [comment]
As discussed with @rich-nguyen and @JustinPinner the omniture script will be inlined, so I will work on this and re-open the PR.

**Comment by jennysivapalan** [comment]
I've inlined the omniture script and will fix the merge conflicts once master is sorted.
@JustinPinner please could you review? I'll revert https://github.com/guardian/frontend/pull/11421 if you're happy.

File: admin/app/pagepresser/HtmlCleaner.scala [hunk 1]
```diff
@@ -10,15 +12,46 @@ object HtmlCleaner {
...
+    document.body().append(omnitureTag)
+    document
   }
 
+  def fetchOmnitureTags(document: Document): String = {
+    val omnitureCode = document.getElementById("omnitureNoScript").getElementsByTag("img").attr("src")
+    val params = parse(omnitureCode).query.paramMap
+
+    val requiredParams: Map[String, Seq[String]] = params.filterKeys(key => List("pageName", "ch", "g", "ns", "c19").contains(key)) ++
+      Map("AQB" -> List("1"), "ndh" -> List("1"), "ce" -> List("UTF-8"), "cpd" -> List("2"), "AQE" -> List("1"), "v14" -> List("D=r"), "v9" -> List("D=g"))
```
Comment (JustinPinner) [inline admin/app/pagepresser/HtmlCleaner.scala]: :open_mouth:

**Comment by JustinPinner** [comment]
I think this looks like it'll work. That Omniture params code isn't very pleasing, but if it gets the job done :ok: Good luck with the merge conflicts :hammer: :+1:

**Comment by jennysivapalan** [comment]
Thanks, the Omniture parameters were defined with Matt K. It was to grab a few from the existing HTML page and then hard code the others. :neutral_face:

**Pull request merged**